commit cfb73280d301d51b3e9b4961c2471063b22a0dab
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-06-26T17:46:14+01:00 (3 months ago)
    
    TOOLS-2283 move manta and smartos jr repos.json out of triton-dev

diff --git a/repos.json b/jr-manifest.json
similarity index 100%
rename from repos.json
rename to jr-manifest.json
